### PR TITLE
Feature/2654 iso8601 as alias for date time

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2654.
No CNR entry ... Right?
2 functests for DateTime copied and changed to be used with "ISO8601"

### CODE REVIEW

File: doc/manuals/user/ngsiv2_implementation_notes.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ Orion always provides datetime attributes/metadata using the format `YYYY-MM-DDT
 timezone (which is the best default option, as clients/receivers may be running in any timezone). This may change in the
```
Comment (fgalan) [inline doc/manuals/user/ngsiv2_implementation_notes.md]: I think the change corresponding to this PR deserves a CNR entry. At the end, it involves a functional change to improve integration with legacy system, so I think is a good idea to mention it.

Comment (kzangeli) [inline doc/manuals/user/ngsiv2_implementation_notes.md]: Fixed in 554844a

**Comment by fgalan** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**